packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -855,7 +865,13 @@ abstract class ClusterBase extends Resource implements ICluster {
       throw new Error(`Invalid cdk8s chart. Must contain a 'toJson' method, but found ${typeof cdk8sChart.toJson}`);
     }
 
-    return this.addManifest(id, ...cdk8sChart.toJson());
+    const manifest = new KubernetesManifest(this, id, {
```

Comment:
Just switched from `addManifest` to `KubernetesManifest` because the former doesn't allow passing options to.